components/match2/wikis/stormgate/match_group_input_custom.lua

```diff
@@ -434,7 +434,6 @@ function CustomMatchGroupInput._mapInput(match, mapIndex, subGroupIndex)
 	map.extradata = {
 		comment = map.comment,
 		header = map.header,
-		displayname = map.mapDisplayName,
```

Comment:
Shouldn't this still happen, as it wouldn't be stored otherwise?

Reply:
That might be an oversight?

Reply:
Oversight either in storing too much or not using it for Stormgate ;)
As it doesn't seem to be needed, remove it for now